DataONEorg/slinky
# Python 3 Upgrade

## Summary
This PR is about upgrading the stack from Python 2.7 to Python 3

## Summary (k8)
The changes here aren't too crazy. The deployment files now point to the right python path; I cheaped out and updated the images in dockerhub so that the `latest` tag already contains the Python 3 images.

The dockerfiles 

## To Test

1. Checkout this repository
2. Checkout this branch
3. Run the kubernetes stack (cd to deploy & run `helm install slinky .`)
4. Expose the service with minikube (`minikube service virtuoso`)
5. Select all triples in virtuoso
6. Ctrl+F for geolink terms
7. Confirm that they've been inserted
